run_benchmark_slurm.sh

```diff
@@ -1,35 +1,30 @@
 #!/bin/bash
 
-timestamp() {
-#  date +"%T" # current time
-  date +"%Y-%m-%d_%H-%M-%S"
-}
+# Source the common functions script
+source scripts/sh_benchmark_utils.sh
 
 # CONFIGFILE="examples/yaml/test_helm_benchmark.yaml"
 CONFIGFILE=$1
-export DOMAINLAB_CUDA_START_SEED=$2
-
-if [ -z "$2" ]
```

Comment:
why these are gone?  we do not know if the user specify the starting seed at all?

Reply:
Added a comment about user seed again